WordPress/src/androidTest/java/org/wordpress/android/support/WPSupportUtils.java

```diff
@@ -818,6 +819,10 @@ public static void scrollIntoView(Integer scrollableContainerID, ViewInteraction
     }
 
     public static void dismissJetpackAdIfPresent() {
+        if (BuildConfig.IS_JETPACK_APP) {
+            return;
+        }
+
```

Comment:
We don't expect a JP banner in JP app, so we can return from this function if we're using JP.